DigitalRain/Raindrops/PerColumnSpaceRaindropFactory.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Xna.Framework;
+
+namespace DigitalRain.Raindrops
+{
+    using Columns;
+
+    public class PerColumnSpaceRaindropFactory : IRaindropFactory
```

Comment:
I do like that you lifted up all this functionality (symbol pool calculations) up one level to the factory. Not a huge fan of this verbose name though lol. It is totally an Ethan named class, and I know you are gonna give me flack for criticizing it!

Also, I noticed this file is located inside the `Raindrops` folder. Maybe we should create a separate `Factories` folder because we now have 100 factories? Just a thought feel free to dismiss it.

Reply:
It's because it creates raindrops with a per-column-space level of granularity.  It's the least ugly name I could think of to describe that lol.

But also, the factories might need a rethink.  There's some duplication between them I don't like.

I thought about a factories directory but I'm not sure that grouping is helpful.  But maybe a folder per factory interface (e.g. IRaindropFactory)